doc/source/data/examples/big_data_ingestion.ipynb

```diff
@@ -243,7 +243,7 @@
     "more machines.\n",
     "\n",
     "To ingest 500GiB of data, we'll set up a Ray Cluster.\n",
-    "The provided :download:`big_data_ingestion.yaml <../big_data_ingestion.yaml>`\n",
+    "The provided [big_data_ingestion.yaml](https://raw.githubusercontent.com/ray-project/ray/master/doc/source/data/big_data_ingestion.yaml)\n",
```

Comment:
Should we use this link or `https://github.com/ray-project/ray/blob/master/doc/source/data/big_data_ingestion.yaml`